models/comments.models.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection");
+
+exports.selectAllCommentsByArticleId = (
+  article_id,
+  sort_by = "created_at",
+  order = "desc"
+) => {
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY ${sort_by} ${order}`,
+      [article_id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
```

Comment:
Have a think about what would happen here if we had a request for an existing article that didn't have any comments - it would give us an error rather than passing us an empty array of comments.